invenio_jsonschemas/views.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio JSONSchemas.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Views provided by Invenio JSONSchemas."""
+
+import datetime
+import os
+import os.path
+import time
+
+from collections import OrderedDict
+
+from wsgiref.handlers import format_date_time
+
+from flask import Blueprint, jsonify, render_template
+
+from flask_breadcrumbs import default_breadcrumb_root, register_breadcrumb
+
+from invenio.base.i18n import _
+
+from invenio.ext.cache import cache
+
+from .api import get_schema_data, get_schemas, internal_schema_url
+
+
+blueprint = Blueprint(
+    'jsonschemas',
+    __name__,
+    url_prefix='/jsonschemas',
+    static_folder='static',
+    template_folder='templates',
+)
+
+default_breadcrumb_root(blueprint, '.jsonschemas')
+
+
+@blueprint.route('/', methods=['GET', 'POST'])
+@register_breadcrumb(blueprint, '.', _('JSON Schemas'))
+def index():
+    """Render schema index with all known schema files."""
+    def _split_path(string):
```

Comment:
can you please move following 3 internal functions to separate files so they can be easily tested?